backstop/tests/typography.html

```diff
@@ -29,20 +29,51 @@
     <h1>Typography</h1>
     <hr />
 
-    <h1 class="iui-text-headline iui-text-spacing">Headline</h1>
+    <label
+      id="skeleton-toggle"
+      class="iui-toggle-switch-wrapper iui-label-on-right"
+      style="margin-bottom: 22px;"
+    >
+      <input
+        class="iui-toggle-switch"
+        type="checkbox"
+        role="switch"
+      />
+      <span class="iui-toggle-switch-label">Skeleton text</span>
+    </label>
+
+    <section id="demo-typography">
+
+      <h1 class="iui-text-headline iui-text-spacing">Headline</h1>
+
+      <h2 class="iui-text-title iui-text-spacing">Title</h2>
+
+      <h3 class="iui-text-subheading iui-text-spacing">Subheading</h3>
 
-    <h2 class="iui-text-title iui-text-spacing">Title</h2>
+      <h4 class="iui-text-leading iui-text-spacing">Leading</h4>
 
-    <h3 class="iui-text-subheading iui-text-spacing">Subheading</h3>
+      <p class="iui-text-block iui-text-spacing">Body</p>
 
-    <h4 class="iui-text-leading iui-text-spacing">Leading</h4>
+      <p class="iui-text-block iui-text-muted iui-text-spacing">Muted</p>
 
-    <p class="iui-text-block iui-text-spacing">Body</p>
+      <p class="iui-text-small iui-text-spacing">Small</p>
 
-    <p class="iui-text-block iui-text-muted">Muted</p>
+      <!-- Paragraph with no content -->
+      <p
+        class="iui-text-spacing"
+        style="width: 127.53px; height: 22px;"
+      ></p>
 
-    <p class="iui-text-small">Small</p>
+    </section>
 
-    <p class="iui-text-block iui-text-spacing iui-skeleton">Lorem ipsum dolor sit amet.</p>
+    <script>
+      document.querySelector('#skeleton-toggle input[type=checkbox]').addEventListener('change', (e) => {
+        document.querySelectorAll('.iui-text-spacing').forEach((skeletonText) => {
+          skeletonText.classList.toggle('iui-skeleton');
+          skeletonText.toggleAttribute("aria-hidden");
+          skeletonText.toggleAttribute("aria-busy");
```

Comment:
Busy shouldn't be used as it is ignored in most screen readers.

Reply:
Removed `aria-busy`